packages/vitest-pool-workers/src/pool/index.ts

```diff
@@ -369,6 +365,16 @@ function buildProjectWorkerOptions(
 		}
 	}
 
+	const { mode } = getNodeCompat(
+		runnerWorker.compatibilityDate,
+		runnerWorker.compatibilityFlags
+	);
+
+	if (mode !== "v1" && mode !== "v2") {
+		runnerWorker.compatibilityFlags.push("nodejs_compat");
+		runnerWorker.compatibilityFlags.push("no_nodejs_compat_v2");
```

Comment:
Pedantic NIT 😄 
```suggestion
		runnerWorker.compatibilityFlags.push("nodejs_compat", "no_nodejs_compat_v2");
```

Reply:
Thanks, I'm not really with it this morning...